### PR TITLE
Cygnus X-1 Testnet instructions part 2

### LINKED ISSUE #593: README for running node

### PR DESCRIPTION
**Pull request by shanev**
Fixes #593

### CODE REVIEW

File: build-genesis.sh [hunk 1]
```diff
@@ -23,7 +23,7 @@ if ! [ -f genesis.json ]; then
 fi
 starsd export-airdrop-snapshot uatom genesis.json snapshot.json
 starsd init testmoniker --chain-id $CHAIN_ID
-starsd prepare-genesis testnet $CHAIN_ID
+starsd prepare-genesis mainnet $CHAIN_ID
```
Comment (JakeHartnell) [inline build-genesis.sh:26]: Seems a bit weird, but I understand why this is. Probably fine.

File: cygnusx-1/part1.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
+# Stargaze Cygnus X-1 Testnet Instructions: Part 1
```
Comment (JakeHartnell) [inline cygnusx-1/part1.md:1]: Do we really need this file?

**Review by JakeHartnell — APPROVED** [review]
LGTM, made a few comments, but I think this is ready.

File: cygnusx-1/part1.md (hunk 2 above)
Comment (shanev) [reply cygnusx-1/part1.md:1]: Just to keep it around for posterity.

File: build-genesis.sh (hunk 1 above)
Comment (shanev) [reply build-genesis.sh:26]: Yeah just to get the same params as mainnet.

**Pull request merged**